lib/suspenders/version.rb

```diff
@@ -1,5 +1,5 @@
 module Suspenders
-  VERSION = "3.0.0".freeze
+  VERSION = "20240516.0".freeze
```

Comment:
@mike-burns in #1106 we moved over to [calver](https://calver.org), but now I'm wondering what the ramifications would be if we switched back to [semver](https://semver.org)?

I ask because I've been refering to this next release as `3.0.0` 😅

Reply:
Release numbers are, like, vibes, ya know?

But since `20230516.0` > `3.0`, I think we're stuck with calver.

Reply:
> But since 20230516.0 > 3.0, I think we're stuck with calver.

Yeah, I was thinking the same thing. Maybe I can give this release a code-name for better branding. Maybe "tailored".

Reply:
Or we can hold the release until the year 3000.

I like your code name.